File: maiworld-seller-interface/src/main/java/com/maiworld/seller/service/CostService.java
package com.maiworld.seller.service;

import java.util.List;

import com.maiworld.pojo.TbCost;

import entity.PageResult;

public interface CostService {

	public List<TbCost> findAll();
	
	public PageResult findPage(int pageNum, int pageSize);
	
	public void add(TbCost cost);
	
	public void update(TbCost cost);
	
	public TbCost findOne(Long id);
	
	public void delete(Long[] ids);
	
	public PageResult findPage(TbCost cost , int pageNum, int pageSize);

	//public List<TbCost> studentList(TbCostExample example);

	public PageResult studentList(int pageNum, int pageSize);
	
//	public List<Map> studentList();
	
}

File: maiworld-seller-web/src/main/webapp/js/service/costService.js
//服务层
		app.service('costService' ,function($http){
			
    				this.findAll=function(){    					
    					return $http.get('../cost/findAll.do');
    				}
    				
    				this.findPage=function(page, rows){
    					return $http.get('../cost/findPage.do?page='+page+'&rows='+rows);
    				}
    				
    				this.findOne=function(id){
    					return $http.get('../cost/findOne.do?id='+id);
    				}
    				
    				this.add=function(entity){
    					return $http.post('../cost/add.do',entity);
    				}
    				
    				this.update=function(entity){
    					return $http.post('../cost/update.do',entity);
    				}
    				
    				this.dele=function(ids){
    					return $http.get('../cost/delete.do?ids='+ids);
    				}
    				
    				this.search=function(page, rows, searchEntity){
    					return $http.post('../cost/studentList.do?page='+page+'&rows='+rows,searchEntity);
    				}
    				
    				this.studentList=function(){					
    					return $http.get('../cost/studentList.do');						
    				}			
    				
    			});